input/pagecontent/group-disease.md

```diff
@@ -42,6 +42,13 @@ Several widely-used terminologies in the cancer domain, including ICD-O-3 and AJ
 
 Under the [Fair Use doctrine](https://www.copyright.gov/fair-use/more-info.html), the IG includes examples illustrating mCODE's representation of cancer diagnoses and AJCC staging values for the purposes of technical implementation guidance to FHIR developers.
 
+### Representing non-TNM Staging Information
+
+Not all cancer types are staged with a TNM-based staging system. Hematological cancers like leukemias, multiple myeloma, and some lymphomas.  Some specialized solid tumors like gynecologic tumors are staged using the FIGO (International Federation of Gynecology and Obstetrics) staging system. Staging systems in this category include Rai, Binet, Cottswold
```

Comment:
```suggestion
Not all cancer types are staged with a TNM-based staging system, including hematological cancers like leukemias, multiple myeloma, and some lymphomas. Some specialized solid tumors like gynecologic tumors are staged using the FIGO (International Federation of Gynecology and Obstetrics) staging system. Other non-TNM staging systems include Rai, Binet, and Cottswold.
```

Related to the non-TNM staging systems list, this is what's in the value set:

```
* NCIT#C141138 "Ann Arbor Stage"
* NCIT#C141212 "Binet Stage"
* NCIT#C125738 "FIGO Stage"
* NCIT#C141147 "Lugano Stage"
```

Should Rai and Cottswold be added?

Reply:
May, please do that if there are NCI codes available.